viewflow_extensions/views.py

```diff
@@ -61,3 +65,8 @@ def get_success_url(self):
         if self._save:
             return self.request.get_full_path()
         return super().get_success_url()
+
+
+def _make_fields_not_required(form):
```

Comment:
No private functions ;) Make it public write a docstring, but it in the proper package ;)

Reply:
and write a unit test :P

Reply:
what do you mean "but it in the proper package"?

Reply:
sorry I meant module. I would recommend `forms` or `utils`.